src/Files.App/UserControls/TabBar/TabBar.xaml.cs

```diff
@@ -51,6 +55,14 @@ public bool AllowTabsDrag
 		public Rectangle DragArea
 			=> DragAreaRectangle;
 
+		private GridLength _titleBarWidth;
+
+		public GridLength TitleBarWidth
+		{
+			get => _titleBarWidth;
+			set => _titleBarWidth = new((value.Value / MainWindow.Instance.GetDpiScale()) + _gap);
```

Comment:
Please see if you can use `App.AppModel.AppWindowDPI`.

Reply:
Thanks, I didn't know.